src/config.cc

```diff
@@ -318,6 +324,67 @@ void Config::initFieldCallback() {
         return srv->storage_->SetDBOption(trimRocksDBPrefix(k),
                                           std::to_string(RocksDB.max_total_wal_size * MiB));
       }},
+      {"rocksdb.enable_blob_files", [](Server* srv, const std::string &k, const std::string& v)->Status {
+        if (!srv) return Status::OK();
+        std::string enable_blob_files = v == "yes" ? "true" : "false";
+        Status s = srv->storage_->SetColumnFamilyOption(Engine::kMetadataColumnFamilyName, trimRocksDBPrefix(k),
```

Comment:
How about using `set_cf_option_cb` to apply the column family option?

Reply:
`set_cf_option_cb` will set ALL cf

Reply:
yes, it would be confusing if some options apply to all column families and some not, To be more clear, what I mean is that maybe we should keep consistent. How do you think?

Reply:
Do you mean to apply this option to all CF?

Reply:
Yes, it should make sense since it wouldn't separate the key-value if the size didn't exceed the blob threshold, right?

Reply:
Another question, those options should NOT take effect if we didn't enable the blob store?

Reply:
Indeed, I will add a judgment here.